app/_data/docs_nav_gateway_3.4.x.yml

```diff
@@ -230,11 +232,26 @@ items:
         items:
           - text: Upgrade Kong Gateway 3.x.x
             url: /upgrade/
-            src: /gateway/upgrade-v2
+          - text: Upgrade Strategies
+            items:
+              - text: Dual-Cluster Upgrade
+                url: /upgrade/dual-cluster/
+              - text: In-Place Upgrade
+                url: /upgrade/in-place/
+              - text: Blue-Green Upgrade
+                url: /upgrade/blue-green/
+              - text: Rolling Upgrade
+                url: /upgrade/rolling-upgrade/
+          - text: Backup and Restore
```

Comment:
I think it might make sense to move this page before the Upgrade Strategies since people would want to do this before they follow the upgrade steps.